projects/kit/components/stepper/stepper.component.ts

```diff
@@ -55,6 +55,12 @@ export class TuiStepperComponent {
         return itemsQueryListObservable(this.steps).pipe(delay(0));
     }
 
+    get stepElements(): HTMLElement[] {
```

Comment:
How about we make this a method `getStepElements`, since this getter creates new instance of an array? I think we can just use `toArray()` since we `map` anyway. Also pay attention not to create public getters unless you need them to be public :)

Reply:
fixed